js/index.js

```diff
@@ -69,4 +69,69 @@ for (var i = 0; i < skills.length; i++) {
     // Append the skill item to the skillsList
     skillsList.appendChild(skill);
 }
+ // Select the #messages section by id
+ var messageSection = document.getElementById("messages");
+
+ // Select the <ul> element within the messageSection
+ var messageList = messageSection.querySelector("ul");
+
+ function updateVisibility() {
+    // Check if the messageList is empty
+    if (messageList.children.length === 0) {
+        messageSection.style.display = "none"; // Hide the messages section
```

Comment:
I prefer using the hidden property rather than changing the display style.  messageSection.hidden = true.  This avoid hard coding the display style.